tests/api/Jalasoft.TeamUp.Projects.API.Tests/PostInvitationTest.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace Jalasoft.TeamUp.Projects.API.Tests
+{
+    using System.IO;
+    using Jalasoft.TeamUp.Projects.API.Controllers;
+    using Jalasoft.TeamUp.Projects.Core.Interfaces;
+    using Jalasoft.TeamUp.Projects.Models;
+    using Jalasoft.TeamUp.Projects.ProjectsException;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Moq;
+    using Xunit;
+
+    public class PostInvitationTest
+    {
+        private readonly Mock<IInvitationsService> mockInvitationsService;
+        private readonly DefaultHttpContext mockHttpContext;
+        private readonly PostInvitation postInvitation;
+
+        public PostInvitationTest()
+        {
+            this.mockInvitationsService = new Mock<IInvitationsService>();
+            this.mockHttpContext = new DefaultHttpContext();
+            this.postInvitation = new PostInvitation(this.mockInvitationsService.Object);
+        }
+
+        [Fact]
+        public async void PostInvitation_Returns_CreateInvitation_Invitation()
```

Comment:
https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices  please follow the naming convention adopted by the team in the other unit tests

Reply:
if any doubt please reach out to @lilian-delgado-jalafund  or @niver-copa-jalafund